### PR TITLE
Update index.html

### PR DESCRIPTION
**Pull request by Jeewal**

### CODE REVIEW

**Comment by Masquerade0097** [comment]
@Jeewal Please fill in the PR description. Also, fix the build.

File: index.html [hunk 1]
```diff
@@ -1,3 +1,22 @@
+Skip to content
```
Comment (akshatnitd) [inline index.html]: These changes are extra and you need to remove it from the Pull Request.

File: index.html [hunk 2]
```diff
@@ -1537,3 +1545,16 @@ <h2>Mario Behling</h2>
 </body>
 
 </html>
+© 2018 GitHub, Inc.
```
Comment (akshatnitd) [inline index.html]: These changes are extra and you need to remove it from the Pull Request.

**Review by akshatnitd — CHANGES_REQUESTED** [review]

**Comment by Jeewal** [comment]
@akshatnitd  Please have a look

**Review by abishekvashok — CHANGES_REQUESTED** [review]
Please fix Travis :)

**Comment by Jeewal** [comment]
https://github.com/fossasia/2012.fossasia.org/pull/176
I hope this can help fix my travis error.

**Comment by Jeewal** [comment]
@abishekvashok @akshatnitd

**Comment by abishekvashok** [comment]
Can I know which issue does this solve?

**Comment by Jeewal** [comment]
The failing of Travis due to the Ruby version being 2.2.0. Whereas, Ruby version 2.3.0 is required

**Comment by abishekvashok** [comment]
The issue in the tracker pls? Travis is fixed

**Comment by Jeewal** [comment]
https://github.com/fossasia/2012.fossasia.org/issues/165

**Comment by Jeewal** [comment]
Has been solved

**Review by Masquerade0097 — APPROVED** [review]
LGTM

**Review by YashKumarVerma — CHANGES_REQUESTED** [review]
1. commits not squashed 
2. vague (too borad) PR Name

**Comment by prateekiiest** [comment]
@Jeewal do you want to open a new PR with the changes

**Pull request closed without merging**